docs/mast/mast.rst

```diff
@@ -324,35 +286,77 @@ Product filtering can also be applied directly to a table of products without pr
                 >>> print(len(products))
                 4
 
+
+Downloading Data Products
+-------------------------
+
+Products can be downloaded by using `~astroquery.mast.ObservationsClass.download_products`,
+with a `~astropy.table.Table` of data products, or a list (or single) obsid as the argument.
+
+.. code-block:: python
+
+                >>> from astroquery.mast import Observations
+
+                >>> single_obs = Observations.query_criteria(obs_collection="IUE",obs_id="lwp13058")
+                >>> data_products = Observations.get_product_list(single_obs)
+
+                >>> manifest = Observations.download_products(data_products, productType="SCIENCE")
+                Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=http://archive.stsci.edu/pub/iue/data/lwp/13000/lwp13058.mxlo.gz to ./mastDownload/IUE/lwp13058/lwp13058.mxlo.gz ... [Done]
+                Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=http://archive.stsci.edu/pub/vospectra/iue2/lwp13058mxlo_vo.fits to ./mastDownload/IUE/lwp13058/lwp13058mxlo_vo.fits ... [Done]
+
+                >>> print(manifest)
+
+                                   Local Path                     Status  Message URL 
+                ------------------------------------------------ -------- ------- ----
+                    ./mastDownload/IUE/lwp13058/lwp13058.mxlo.gz COMPLETE    None None
+                ./mastDownload/IUE/lwp13058/lwp13058mxlo_vo.fits COMPLETE    None None
+
+​As an alternative to downloading the data files now, the ``curl_flag`` can be used instead to instead get a curl script that can be used to download the files at a later time.
+
+.. code-block:: python
+
+                >>> from astroquery.mast import Observations
+
+                >>> single_obs = Observations.query_criteria(obs_collection="IUE",obs_id="lwp13058")
+                >>> data_products = Observations.get_product_list(single_obs)
+                
+                >>> Observations.download_products(data_products,
```

Comment:
`download_products` returns a table, maybe it's worth assigning it into a variable. However, it stays as is (I really don't have a strong preference), then include the table in the output below.